packages/discovery-provider/src/api/v1/models/activities.py

```diff
@@ -10,79 +8,91 @@
 )
 from .tracks import track, track_full
 
+base_activity_model = ns.model(
+    "activity",
+    {
+        "timestamp": fields.String(required=True),
+        "item_type": fields.String(enum=["track", "playlist"], required=True),
+        "item": fields.Raw(required=True),
```

Comment:
cc @rickyrombo I ended up leaving the `item` field on the base class here so that the generated `Activity` type keeps its `item` field (as an object). But if the discriminator field exists, it will parse it as the more specific type.